tox.ini

```diff
@@ -21,17 +21,19 @@ envlist =
     {py27,py35,py36,py37,py38,py39,py310}-profile{,-gevent}
     {py27,py35,py36,py37,py38,py39,py310}-profile-minreqs{,-gevent}
     py{37,38,39,310}-profile-protobuf319
+    py{27,35,36,37,38,39,310,311}-tracer_test_http
```

Comment:
Worth having a comment here about why this suite exists (the work-around for riot issue). Maybe `FIXME[riot-3.11]`